state/address.go

```diff
@@ -353,11 +353,11 @@ type address struct {
 type Origin string
 
 const (
```

Comment:
I'd consider relocating this to core. We should probably expose it in the other core types eventually.